_pkgdown.yml

```diff
@@ -27,6 +27,7 @@ reference:
       - concat
       - dev_suppress
       - eval_code
+      - replace_code
```

Comment:
Internal docs shouldn't be listed in pkgdown.yml, please remove

Reply:
The one I always forget :roll_eyes: